spec/factories/projects.rb

```diff
@@ -32,6 +34,14 @@
       private(true)
     end
 
+    factory :approved_project do
```

Comment:
This and the live_project factory seem unnecessary since you're already setting them to true in the default project factory.